src/css.js

```diff
@@ -58,12 +56,11 @@ function vendorPropName( name ) {
 	}
 }
 
-function setPositiveNumber( elem, value, subtract ) {
-	var matches = rnumsplit.exec( value );
```

Comment:
Yeah, it looks like we're missing some for setting `border` and `padding` to negative values.